### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by RDIL**
New/updated examples

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -263,6 +263,17 @@ build_package_test_task:
...
 ```
 
+### Linting
+
+You can easily set up linting with Cirrus CI and flake8, here is an example:
+
+```yaml
+lint_task:
+  container:
+    image: jumbocakeyumyum/flake8:latest
```
Comment (fkorotkov) [inline docs/examples.md]: Can you use the official `alpine/flake8:latest` image?

Comment (RDIL) [reply docs/examples.md]: oh didn't know that existed, I used my own image 😅

Comment (fkorotkov) [reply docs/examples.md]: Seems legit: https://hub.docker.com/r/alpine/flake8

Comment (RDIL) [reply docs/examples.md]: wait no this cant be legit it says `...docker build for fake8 via travis ci`

Comment (RDIL) [reply docs/examples.md]: if I remember correctly this isn't travis 😆

**Pull request merged**